.github/workflows/phpunit-tests-run.yml

```
@@ -161,6 +161,10 @@ jobs:
         if: ${{ inputs.multisite }}
         run: node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit --verbose -c ${{ env.PHPUNIT_CONFIG }} --group ms-files
 
+      - name: Run tests as a subdomain multisite install
+        if: ${{ inputs.multisite }}
+        run: node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit --verbose -c ${{ env.PHPUNIT_CONFIG }} -d SUBDOMAIN_INSTALL=true
```

I think it makes sense to run this in parallel to avoid multisite jobs from taking double the amount of time. The best way to do this would likely be to add a new input for `multisite-type` that defaults to `subdirectory` and change this condition to something like:

```suggestion
        run: node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit --verbose -c ${{ env.PHPUNIT_CONFIG }}${{ 'subdomain' == inputs.multisite-type && ' -d SUBDOMAIN_INSTALL=true' || '' }}
```

Probably should make the `ms-files` job dynamic as well.

Working this into the matrix in `phpunit-tests.yml` is a little tricky,. We should be able to add a `multisite-type: [ 'subdomain', 'subdirectory' ]` accompanied with an `exclude:` excluding all `multisite: false/multisite-type: 'subdomain` and `multisite: false/multisite-type: 'subdirectory` jobs.